develop/dev-guide-proxysql-integration.md

```diff
@@ -624,7 +624,7 @@ The following steps will set up ProxySQL and TiDB on ports `6033` and `4000` res
 
 ## Production environment
 
-For a production environment, it is recommended that you use [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) directly for a fully-managed experience.
+For a production environment, it is recommended that you use [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) directly for a fully-managed experience.
```

Comment:
It jumps to https://www.pingcap.com/tidb-dedicated/. And the following steps are based on TiDB Dedicated.
```suggestion
For a production environment, it is recommended that you use [TiDB Dedicated](https://www.pingcap.com/tidb-dedicated/) directly for a fully-managed experience.
```